src/scripts/main.js

```diff
@@ -354,7 +354,35 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the '.dashboard' selector actually exists in the DOM before attempting to append children to it. If it's possible that the element may not exist, you should add a conditional check to prevent potential errors.

Reply:
fix it, just add if